HopsanGUI/LogVariable.cpp

```diff
@@ -2134,6 +2134,7 @@ void createBodeVariables(const SharedVectorVariableT pInput, const SharedVectorV
 
     // Create the output variables for bode diagram
     SharedVectorVariableT pFrequencyVar = createFreeFrequencyVectorVariabel(freq);
+    pFrequencyVar->multData(M_PI*2);
```

Comment:
Could have multiplied directly when creating freq, but maybe it does not matter. The createFreeFrequencyVariable function defaults to rad/s, so the assumtion us that the input data is in this format